_posts/blog/2019-02-28/2019-02-28-how-to-spend-bitcoin-and-other-cryptocurrencies-on-openbazaar.markdown

```diff
@@ -0,0 +1,73 @@
+
+<br>  
+
+#### 3. Moderator Types to Make Choosing a Moderator Easier
+
+![OB1 Verified Moderator Shield on OpenBazaar](OB1_Verified_Moderator_Shield.gif "OB1 Verified Moderator Shield on OpenBazaar")
+
+Anyone can decide to become a moderator on OpenBazaar so it’s very important to choose someone who is trustworthy. You may notice that some listings have an orange shield on them which signals that the seller has provided buyers with a choice of OB1 Verified Moderators. An OB1 Verified Moderator has submitted some reputation & identity information to OB1, the company leading the development of OpenBazaar, to demonstrate they are more likely to be trustworthy. 
+
+If a seller does not have moderators or [OB1 Verified Moderators](https://ob1.io/verified-moderators.html) listed on their item but you would like to use one on your transaction, just message the seller to request it. If they decline, proceed with caution unless you know and trust them and are willing to risk your coins.
+
+<br>  
+
+#### 4. Different Listing Types So You Can Easily Search for Items, Services, Digital Goods and Cryptocurrencies
+
+![Listing Types on OpenBazaar for Services, Cryptocurrencies, Physical Goods or Digital Goods](OpenBazaar_Listing_Type.gif "Listing Types on OpenBazaar for Services, Cryptocurrencies, Physical Goods or Digital Goods")
+
+We often use the word “thing” to describe what’s for sale on OpenBazaar but did you know that people can also sell things that are not just _physical_ things? Since its inception OpenBazaar users have listed “things” you may think of right away like books, clothing and art, but also “things” like real estate, internet hosting space and temporary rentals...the last two of which start to blur the line between “thing” and “service.”
+
+Services are also a distinct listing type available in OpenBazaar. People have listed services like translation, design, coding or other business services. Even ridesharing services! (Wait, is that also on the blurry line of service/thing too?)
+
+Another listing type available is digital goods like ebooks, music, stock photos, game codes, gift cards and more. These items are particularly popular because they don’t require anyone to wrestle with the sometimes complicated logistics of shipping physical items around the world.
```

Comment:
I would add something like, "Digital goods also don't require you to share your identity with the other party. You can buy and sell as many digital items as you like without ever sharing your identity information."